doc/api/cli.md

```diff
@@ -832,6 +832,16 @@ unless either the `--pending-deprecation` command-line flag, or the
 are used to provide a kind of selective "early warning" mechanism that
 developers may leverage to detect deprecated API usage.
 
+### `--policy-deny-fs`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+TODO
```

Comment:
When you get to filling out this piece, be sure to include details on what internal functions are impacted by disabling filesystem access and how to mitigate those (e.g. repl history, module loading, etc)